app/Http/Controllers/admin/StatisticsController.php

```diff
@@ -4,6 +4,15 @@
 
 include __DIR__ . '/../../../../resources/views/admin/statistics/SampleData.php';
 
+/* Following namespace import statement will be used at the phase of backend
+use Apps\Models\User;
+use Apps\Models\Reservation;
+use Apps\Models\Attribute;
+*/
+namespace App\Http\Controllers\admin;
+
+include __DIR__ . '/../../../../resources/views/admin/statistics/SampleData.php';
```

Comment:
Can make this to decrease ../? in the future maybe this will cause some problems.

Reply:
@keanbillyfernandez 
`include __DIR__ . '/../../../../resources/views/admin/statistics/SampleData.php';`
This will be removed in the end since it's sample for creating UI.
Thank you for checking.